database/migrations/000003_add_thumbnail_column.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE videos
+ADD COLUMN IF NOT EXISTS thumbnail TEXT;
```

Comment:
Add an endline character.

```suggestion
ADD COLUMN IF NOT EXISTS thumbnail TEXT;

```